### PR TITLE
meson: generate and install trust-extract-compat

### PR DESCRIPTION
**Pull request by xry111**
Now meson.build files don't mention trust-extract-compat.  This PR makes meson generate and install it, just like autotools.

### CODE REVIEW

File: trust/meson.build [hunk 1]
```diff
@@ -105,6 +105,11 @@ executable('trust',
                           libp11_tool_dep] + libffi_deps + dlopen_deps + libtasn1_deps,
            install: true)
 
+configure_file(input: 'trust-extract-compat.in',
+               output: 'trust-extract-compat',
+               configuration: configuration_data(),
```
Comment (ueno) [inline trust/meson.build:110]: It's not a problem in your patch, but it looks like `trust-extract-compat.in` doesn't contain any substitution keywords (i.e., @...@). Maybe we can just rename `trust-extract-compat.in` to `trust-extract-compat` in the repo?

**Review by ueno — COMMENTED** [review]
Good catch, thank you!

File: trust/meson.build (hunk 1 above)
Comment (xry111) [reply trust/meson.build]: I agree.  New version force-pushed.

**Comment by ueno** [comment]
To fix the CI failure, I guess you would need to add `EXTRA_DIST += trust/trust-extract-compat` in `trust/Makefile.am`.

**Comment by ueno** [comment]
Thanks!

**Pull request merged**